build/scripts/gofmt.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh	
+
+out=$(gofmt -l -s **/*.go | grep -v vendor)
```

Comment:
If we intend to run gofmt on every .go file, this pattern ignores go files inside subfolder (e.g apiserver/apiserver/cli.go)